src/modules/fetcher/__tests__/FetcherModule.spec.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import FetcherModule from '../FetcherModule'
+import NativeFetcherModule from '../NativeFetcherModule'
+import type { TargetFilePathsType } from '../FetcherModule'
+
+jest.mock('../NativeFetcherModule')
+
+beforeEach(() => {
+  jest.resetAllMocks()
+})
+
+describe('FetcherModule', () => {
+  const testTargetFilePaths: TargetFilePathsType = {
+    'local/path/to/resource.png': 'http://randomtesturl.de/resource.png',
+    'local/path/to/resource2.jpg': 'http://randomtesturl.de/resource2.jpg'
+  }
+
+  it('currentlyFetching should be false if no fetching process is running', () => {
```

Comment:
`currentlyFetching` is an implementation detail to me and should not be tested.

Reply:
Only the effects of fetching twice should be tested.

You can do this by fetching twice and not resolving for one call.